src/layouts/header/Header.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { PROFILE_DATA } from "../../apis/api";
+import "./Header.css";
+
+const Header = () => {
+  const [profileData, setProfileData] = useState(null);
+
+  const GetProfileData = async () => {
+    const response = await PROFILE_DATA();
+    if (response === null) {
+      setProfileData(null);
+    } else {
+      const data = response.data;
+      setProfileData({ ...data });
+    }
```

Comment:
@youngjin-korea 

[제안]

에러 핸들링은 호출하는 쪽에서 컨트롤 해야 합니다.

```suggestion
    try {
      const response = await PROFILE_DATA();
      setProfileData({ ...response });
      
      // 정상 응답값 핸들링
    } catch (err) {
      // error 핸들링
    }
```